server/src/graql/gremlin/spanningtree/graph/Node.java

```diff
@@ -18,16 +18,20 @@
 
 package grakn.core.graql.gremlin.spanningtree.graph;
 
+import com.google.common.collect.Sets;
 import grakn.core.graql.gremlin.fragment.Fragment;
 
+import java.util.Arrays;
 import java.util.HashSet;
+import java.util.Objects;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 /**
  * An node in a directed graph.
  *
  */
-public class Node {
+public class Node implements Comparable {
```

Comment:
Why do we need to implement Comparable? is overriding `hashCode` and `equals` not enough?

Reply:
You're right this isn't actually needed anymore